include/librealsense2/hpp/rs_aus.hpp

```diff
@@ -74,6 +74,26 @@ namespace rs2
         return results;
     }
 
+    inline  std::vector<uint8_t> aus_receive_json_data()
+    {
+        std::vector<uint8_t> results;
+
+        rs2_error * e = nullptr;
+        const rs2_raw_data_buffer * buf = rs2_aus_receive_json_data(&e);
+        error::handle(e);
+
+        std::shared_ptr<const rs2_raw_data_buffer> list(buf, rs2_delete_raw_data);
```

Comment:
Can you explain the flow here?
Take the BJSON data, convert to list and then copy a vector?

Reply:
Can we return a real json string like here?
rs2_serialize_json

Reply:
I used vector<uint8_t> because we wanted to use a struct that already exsisted - the raw_data_buffer and add a few func for AUS use, we can change it all to string but then we will need to add delete func, get func and new struct similar to the raw_data_buffer.

Reply:
```
/* Serialize JSON content, returns ASCII-serialized JSON string on success. otherwise nullptr */
rs2_raw_data_buffer* rs2_serialize_json(rs2_device* dev, rs2_error** error);
```

Reply:
Changed